apps/shelter-web/vite.config.mts

```diff
@@ -25,6 +25,12 @@ export default defineConfig(({ mode }) => {
     root: __dirname,
     cacheDir: '../../node_modules/.vite/apps/shelter-web',
 
+    define: {
+      'import.meta.env.VITE_SHELTER_BASE_PATH': JSON.stringify(
+        process.env.SHELTER_BASE_PATH || '/'
+      ),
+    },
+
```

Comment:
@vecchp adding this makes `process.env.SHELTER_BASE_PATH` available as `VITE_SHELTER_BASE_PATH` inside the app (tsx)

so inside `main.tsx` we now have:
```javascript
const basename = import.meta.env.VITE_SHELTER_BASE_PATH || '/';
```

to build locally I run:
`export SHELTER_BASE_PATH=/hello && yarn nx build shelter-web --configuration=production`